.travis.yml

```diff
@@ -1,15 +1,30 @@
 language: node_js
 node_js:
-  - "4.6"
-  - "6.7"
+  - '4.6'
+  - '6.7'
 services:
   - mongodb
 cache:
   directories:
-  - node_modules
+    - node_modules
 git:
   depth: 3
+skip_cleanup: true
 before_script:
+  - echo -e "Host $DEPLOY_HOST\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
   - npm prune
+  - chmod +x deploy.sh
 after_script:
   - npm run report-coverage
+after_success:
+  - ./deploy.sh
+addons:
+  apt:
+    packages:
+      - sshpass
+env:
+  global:
+    - secure: t85HMckHA0Xl0jCXPtkId45JdskkuPFY1yb5CkBM5Jmyb+Pnetwy3pNv+adV65McgiI6CC931KNXC92DjorqNjU96WSBagZLGU/W0nLWGN2TDRXnUEaTJwSNpWnuCy8Le93+3+8hrdgrZ02O/ZTCyKy2RNw43DZs1yDXm5nR5trCqJQKy6rOETgV9EFWcZAodlCM3dICETdkeSRLSc8ygIH2n3cLPTIFDLik+cEik6s+18SBBUaPq2fQWDIWK+ulW5MW/fzdlyij/Bq6kDYifQjZlDvg4MaEjwT5g6RNPW87vs5Lp93hNBWnUSV7hSDLL9qhPCOXyb6/iRXHgexcDI41ya0qyGcfRnsbUb7qz4iI16p+LOsTgGVqx8uYvX9DIGQBsJz7Klpu+4HGJkHBsdyy7Cnt5TxvWNlmvq57lpymITIhy34C2hGLP5NVMyFHvOHLuZCuTdbLAplVXmY/1KGXTFm5ohI7ozLnsb5JbPoi/oh00CwaVm4RF65Ehi1Jr8/8nj5D8MYFj5yiYwxjhct/eOTGluTR3Gv47wrH7DLzqUn3Hr4+mLuRbIqiXfnrvVXCsqZybtf8XXsPASsf2Dx6rEdBLIiJDhzUgDKEa03pp/NneDcYJ0NBHKovDN4Y30BvIApnFN7TE18n3q0s8f4AUJpAULYoj+2JtZuJ8bY=
```

Comment:
It's pretty standard to have the travis do the [encryption](https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml) for you and put it in your .travis.yml file.